backend/src/main/java/wooteco/retrospective/dto/AttendanceResponse.java

```diff
@@ -0,0 +1,35 @@
+package wooteco.retrospective.dto;
+
+import wooteco.retrospective.domain.attendance.Attendance;
+import wooteco.retrospective.domain.attendance.Time;
+import wooteco.retrospective.domain.member.Member;
+
+public class AttendanceResponse {
+
+    private final String date;
+    private final Member member;
+    private final Time time;
+
+    public AttendanceResponse(String date, Member member, Time time) {
+        this.date = date;
+        this.member = member;
+        this.time = time;
+    }
+
+    public static AttendanceResponse of(Attendance attendance) {
+        return new AttendanceResponse(attendance.getDate(), attendance.getMember(), attendance.getTime());
```

Comment:
```suggestion
        return new AttendanceResponse(
                attendance.getDate(), 
                attendance.getMember(), 
                attendance.getTime()
        );
```

개인적으로 조금 길다싶은 코드는 모두 내려저 정리해주는편 이에요! 사람마다 다르니 참고만 해주시고 그래도 100줄이 넘는 코드는 모두 내려주세요!